doc/guides/device_mgmt/smp_protocol.rst

```diff
@@ -0,0 +1,260 @@
+    | ``2``         | write request                                 |
+    +---------------+-----------------------------------------------+
+    | ``3``         | write response                                |
+    +---------------+-----------------------------------------------+
+
+
+.. _mcumgr_smp_protocol_group_ids:
+
+Management ``Group ID``'s
+=========================
+
+The SMP protocol supports predefined common groups and allows user defined
+groups. Below table presents list of common groups:
+
+
+.. table::
+    :align: center
+
+    +---------------+-----------------------------------------------+
+    | Decimal ID    | Group description                             |
+    +===============+===============================================+
+    | ``0``         | :ref:`mcumgr_smp_group_0`                     |
+    +---------------+-----------------------------------------------+
+    | ``1``         | :ref:`mcumgr_smp_group_1`                     |
+    +---------------+-----------------------------------------------+
+    | ``2``         | :ref:`mcumgr_smp_group_2`                     |
+    +---------------+-----------------------------------------------+
+    | ``3``         | Application/system configuration              |
+    |               | (currently not used by Zephyr)                |
+    +---------------+-----------------------------------------------+
+    | ``4``         | Application/system log management             |
+    |               | (currently not used by Zephyr)                |
+    +---------------+-----------------------------------------------+
+    | ``5``         | Run-time tests                                |
+    |               | (unused by Zephyr)                            |
+    +---------------+-----------------------------------------------+
+    | ``6``         | Split image management                        |
+    |               | (unused by Zephyr)                            |
+    +---------------+-----------------------------------------------+
+    | ``7``         | Test crashing application                     |
+    |               | (unused by Zephyr)                            |
+    +---------------+-----------------------------------------------+
+    | ``8``         | :ref:`mcumgr_smp_group_8`                     |
+    +---------------+-----------------------------------------------+
+    | ``9``         | :ref:`mcumgr_smp_group_9`                     |
+    +---------------+-----------------------------------------------+
+    | ``63``        | Zephyr specific basic commands group          |
+    +---------------+-----------------------------------------------+
+    | ``64``        | this is the base group for defining           |
+    |               | an application specific management groups     |
+    +---------------+-----------------------------------------------+
+
+The payload for above groups, except for ``64`` which is not defined,
```

Comment:
Is it not groups 64 and above? Would add a note here about that

Reply:
done